### PR TITLE
do not use deprecated text for unstable docs

### PR DESCRIPTION
**Pull request by euclio**
# Before

# After

### CODE REVIEW

File: src/librustdoc/clean/mod.rs [hunk 1]
```diff
@@ -2890,7 +2890,8 @@ pub struct Stability {
     pub feature: String,
     pub since: String,
     pub deprecated_since: String,
-    pub reason: String,
+    pub deprecated_reason: String,
```
Comment (euclio) [inline src/librustdoc/clean/mod.rs]: Any reason (no pun intended) these shouldn't be `Option`s?

Comment (brson) [inline src/librustdoc/clean/mod.rs]: Seems like they should be, but that can be done in a follow up.

**Comment by brson** [comment]
@bors r+

Nice fix.

**Comment by alexcrichton** [comment]
@bors: retry force clean

* restarted buildbot

**Pull request merged**